src/pages/Events/Events.tsx

```diff
@@ -31,6 +31,8 @@ const Events = ({ eventData }: EventsProps) => {
     return isAfter(event.date, currentDate);
   });
 
+  filterByDate.sort((a, b) => compareAsc(a.date, b.date));
```

Comment:
Can you put this after the `filteredSearch` and make this sort the `filteredSearch` instead?

It makes more sense to do the sorting once we have the events we are 100% going to display. Currently, we are sorting and then filtering and removing events so they don't really need to be sorted.

Reply:
👍🏻